README.md

```diff
@@ -1,22 +1,49 @@
-
-There's one plugin you need, which is the Lombok plugin. This can be found in IntelliJ's repositories and should be installed before attempting to run the applcation. Next to that, there's one checkbox you need to enable in IntelliJ's settings. Go to `Settings->Build,Execution,Deployment->Compiler->Annotation Processors` and enable the `Enable Annotation Processors` checkbox. 
+Most of the developers (if not all) use IntelliJ, the Java IDE from Jetbrains.
+It's highly recommended, even when you think Eclipse is the best thing out there.
+For this application, you need the Ultimate version, as we need the Spring Boot support.
+Get it here: https://www.jetbrains.com/idea/.
+If you already have IntelliJ installed, make sure you have the latest version (2016.1 at the time of writing).
+We assume you have a working version (including it running on Java 8) before you read on.
+
+There's one plugin you need, which is the Lombok plugin.
+This can be found in IntelliJ's repositories and should be installed before attempting to run the applcation.
+Next to that, there's one checkbox you need to enable in IntelliJ's settings.
+Go to `Settings->Build,Execution,Deployment->Compiler->Annotation Processors` and enable the `Enable Annotation Processors` checkbox.
 
 ### Running the application
 IntelliJ has some neat functionality to make running the application very easy. 
 
 #### Set the properties
-The API has a number of properties that need to be set before the applicaiton can run. There's a template in `resources\config\application.properties.sample`. Copy the contents from this file into a `application.properties`. This file is gitignored because it contains private information such as possible database credetials and payment keys. In the near future, this process will be easier. The sample proprties assume a working PostgreSQL installation running in the background. 
+The API has a number of properties that need to be set before the applicaiton can run.
+There's a template in `resources\config\application.properties.sample`.
+Copy the contents from this file into a `application-dev.properties`.
+This file is gitignored because it contains private information such as possible database credetials and payment keys.
+The sample proprties assume a working PostgreSQL installation running in the background.
 
 #### Spring Boot
-From the Run/Debug Configuration window, add a new Spring Boot configuration. It needs only two settings to run. The *Main class* should be set to **ch.wisv.areafiftylan.Application** and the *Use classpath of mod...* should be set to your project module, which is the only selectable option in nearly all cases. 
+From the Run/Debug Configuration window, add a new Spring Boot configuration.
+It needs only two settings to run.
+The *Main class* should be set to **ch.wisv.areafiftylan.Application** and the *Use classpath of mod...* should be set to your project module, which is the only selectable option in nearly all cases.
```

Comment:
I can't seem to find the `Use classpath of mod...`. Please add the same description type as above on how to find the window/setting.